relay_test.go

```diff
@@ -831,8 +834,7 @@ func TestRelayStalledConnection(t *testing.T) {
 		AddLogFilter("Dropping call due to slow connection.", 1, "sendChCapacity", "32").
 		SetSendBufferSize(32). // We want to hit the buffer size earlier, but also ensure we're only dropping once the sendCh is full.
 		SetServiceName("s1").
-		SetRelayOnly().
-		SetCheckFramePooling()
```

Comment:
is this removal on purpose?

It's a bit surprising given the diff is expected to make the tests "less leaky", right?